src/client/components/line-chart-tooltip/line-chart-tooltip.tsx

```diff
@@ -59,6 +60,10 @@ function measureLabel(dataset: Dataset, range: PlywoodRange, series: ConcreteSer
     formatter={formatter} />;
 }
 
+function hasSecondSplit(essence: Essence): boolean {
```

Comment:
`essence.splits` and its shape is a public API. I think functions like this should be local. Otherwise we will bloat `Essence` class with every little helper.